### DISCUSSION TITLE
Transformer encoder is much slower that python onnx runtime

### DISCUSSION THREAD

**Discussion by igor-yusupov** [sonos/tract#1085] [General]
Is it OK that transformer encoder is much slower than python onnx runtime?

**Comment by igor-yusupov** [comment]
but with version "0.19.8" it works much faster as a python onnx!

**Comment by kali** [comment]
0.20 is slower than 0.19 ? can you give me a model ?

**Comment by igor-yusupov** [comment]
@kali https://pytorch.org/docs/1.13/generated/torch.nn.Transformer.html

or do you need weights and example code?
